Src/GoogleApis.Tests/Apis/Download/MediaDownloaderTest.cs

```diff
@@ -28,10 +28,7 @@
 using NUnit.Framework;
 
 using Google.Apis.Download;
-using Google.Apis.Json;

```

Comment:
Bad merge here as well.... and I'm pretty sure in the other files as well... Take a look in your diff and make sure you don't delete code...